sim/dex.js

```diff
@@ -752,11 +772,41 @@ class ModdedDex {
 				const bans = banNames.map(v => toId(v));
 
 				if (checkTeam) {
-					ruleTable.complexTeamBans.push([innerRule, '', limit, bans]);
+					let index = -1;
```

Comment:
This would be way more readable as a `findIndex` call.

Reply:
Updated.